samples/start-app/php-symfony/migrations/Version20230320134850.php

```diff
@@ -10,7 +10,7 @@
 /**
  * Auto-generated Migration: Please modify to your needs!
  */
-final class Version20230125112208 extends AbstractMigration
+final class Version20230320134850 extends AbstractMigration
```

Comment:
I dont see downside to keep old migrations and add new

Reply:
Theoretically I don't either. But the console migrate script by Symfony deleted the previous file. I was a bit surprised that it did. Maybe I used it wrong... I will try to retrieve it through Github